eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/json/EclairModels.scala

```diff
@@ -136,14 +143,20 @@ case class RelayedPayment(
 )
 
 case class SentPayment(
-    amount: MilliSatoshis,
-    feesPaid: MilliSatoshis,
     paymentHash: Sha256Digest,
-    paymentPreimage: String,
-    toChannelId: FundedChannelId,
-    timestamp: FiniteDuration
+    paymentPreimage: String
 )
 
+object SentPayment {
+  case class Part(
+      id: String,
```

Comment:
should this be `id: UUID` ?

Reply:
Why?

Reply:
It seems that every other `id` in eclair is a UUID.

However I deleted this `object SentPayment` in src and it is only used in one other place -- json serializers. Why do we have this?